msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -803,25 +803,136 @@ Delegated permissions allow the app to access data on behalf of the signed-in us
 
 ## extensions
 
-**Optional**—array
+**Optional**--object
 
-Contains objects that define the set of extensions for the app. Used to specify Outlook Add-ins within an app manifest for simplified distribution and acquisition within the Microsoft 365 ecosystem.
+The set of extensions for this app. Used to specify Outlook Extensions (Add-ins) within an app manifest for simplified distribution and acquisition across the Microsoft 365 ecosystem.
 
 > [!NOTE]
-> Only one extension per app is supported.
+> Currently only one extension per app is supported.
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`requirements`| object | | | Specifies the set of client or host requirements for the extension. |
+|`runtimes`| array | | | Configures the sets of runtimes and actions that can be used by each extension point.  |
+|`ribbons`| array | | | Defines the ribbons extension point. |
+|`autoRunEvents`| array | | | Defines event-based activation extension points. |
+|`prefer`| array | | | Specifies relationship to alternate existing Microsoft 365 solutions. Used for de-duping and backwards compatibility. |
+|`audienceClaimUrl`| string | 2048 characters | | The URL for your extension. Used to validate Exchange user identity tokens. |
+
+For more information, see [Office Add-ins manifest for Microsoft 365](/office/dev/add-ins/develop/unified-manifest-overview) documentation.
+
+### extensions.requirements
+
+Specifies the set of client or host requirements for an extension. If a client doesn't support the specified requirements, the extension will not be available in that client. Requirements are supported at the element and sub-element level. In both cases, the service only returns items that match the host.
```

Comment:
@erikadoyle request you to let us know the required changes here.

Reply:
Being JSON, instead of "element" and "sub-element" we should probably say "object" and "child object".

@Rick-Kirkham, could you help with a better description of "service"? This was taken from Paul's spec ("Outlook Extensions Manifest Element Def"); I don't have further context or expertise.